workflows/transcriptomics/brew3r/BREW3R-tests.yml

```
@@ -0,0 +1,29 @@
+- doc: Test outline for BREW3R
+  job:
+    Input gtf:
+      class: File
+      path: test-data/input.gtf
+      filetype: gtf
+    BAM collection:
+      class: Collection
+      collection_type: list
+      elements:
+      - class: File
+        identifier: GSM4196550
+        path: test-data/GSM4196550_subset.bam
+      - class: File
+        identifier: GSM4196551
+        path: test-data/GSM4196551_subset.bam
+      - class: File
+        identifier: GSM4196552
+        path: test-data/GSM4196552_subset.bam
+    strandedness: 'stranded - reverse'
+    minimum coverage: '10'
+    minimum FPKM for merge: '1.0'
```

```suggestion
    minimum coverage: 10
    minimum FPKM for merge: 1.0
```
did you get this through test generation or did you add this manually ? It doesn't matter for tests now, but I hope that in the future we can check the types :)

---

I did `planemo workflow_test_init --from_invocation` and the parameters format was super strange:
```
    strandedness: stranded - reverse
    minimum coverage: '10'
    minimum FPKM for merge: '1.0'
```
I thought it would be better to put quotes around the strandedness and I wonder what to do with the float/int values.

---

yaml infers types as best as it can, so the quotes are not necessary if the only type it can be is a string.
quotes around native yaml types however turn those values into strings.

```yaml
a: 1  # int
b: "1"  # string
c: a string  # string
d: "a string"  # string, same as c
e: "this: must have quotes"  # without quotes this would be a syntax error. 
```

---

Thanks